server/api/apointment_handler.py

```diff
@@ -0,0 +1,62 @@
+from flask import jsonify, Blueprint, request
+import json
+from model.model import Users, Appointments
+from config import db
+from utils.auth.middleware import check_token
+import dateutil
+import re
+
+create_appointment_handler = Blueprint('create_appointment_handler', __name__)
+get_appointment_handler = Blueprint('get_appointment_handler', __name__)
+
+
+@create_appointment_handler.route('/appointment/create', methods=['POST'])
```

Comment:
i think in the FE branch the route was just /appointment

Reply:
oh, right. I'll change the FE later